blueocean-core-js/.eslintrc

```diff
@@ -5,6 +5,7 @@
     "react/jsx-indent-props": 0,
     "no-unused-vars": [2, {"varsIgnorePattern": "^React$"}],
     "max-len": [1, 160, 4],
+    "prefer-template": 1,
```

Comment:
I figured, seeing the rest of it. totally agree: hate template requirement in jsx. 👍